src/hb-ot-layout.cc

```diff
@@ -62,18 +62,50 @@
  * kern
  */
 
+/**
+ * hb_ot_layout_has_kerning:
+ * @face: The #hb_face_t to work on
+ *
+ * Tests whether a face includes any kerning data. Examines the 'kern' table.
```

Comment:
I don’t think “any” is appropriate here as it does not check `GPOS` kerning. I think something like “includes kerning data in the `kern` table” would be less confusing.